src/runner.ts

```diff
@@ -61,7 +61,7 @@ const runCypress = async (cypressCmd: string, resultsFolder: string, reportsFold
 		await exec([
 			cypressCmd,
 			'--reporter mochawesome',
-			`--reporter-options reportDir="${resultsFolderAbs}",overwrite=false,html=false,json=true`,
+			`--reporter-options reportDir="${resultsFolder}",overwrite=false,html=false,json=true`,
```

Comment:
I'm not sure in the correctness of this, because right now we don't really make any guarantees that the relative path will always work. It depends on which directory we are in, and it's set to whatever the current directory is. To be on the safe side, I would use another solution. It's also weird that absolute paths work everywhere but in this one place. So please consider something else. For example, I found the following working both on Linux and Windows:

```typescript
	const resultsFolderAbs = path.resolve(config.projectPath, resultsFolder)
		.replace(/\\/g, '/');
```